cfbs/commands.py

```diff
@@ -401,19 +402,47 @@ def _someone_needs_me(this) -> bool:
     return 0
 
 
-@commit_after_command("Updated all modules")
-def update_command():
-    new_deps = []
-    new_deps_added_by = dict()
+def update_command(to_update):
```

Comment:
I'd prefer keeping the decorator in place. It guarantees that the code in it will run when the function returns. Having the code at the end of the function means every early `return` will skip it. We may need to extend the decorator to support getting the arguments for the commit message from the return tuple of the decorated function. So the function would do `return (0, list(updated_modules))` and the decorator would use that for the commit message.

Reply:
And make it be a `namedtuple`, maybe? I don't quite care, TBH.